### PR TITLE
feat(component-store): initial setup

### PR DESCRIPTION
**Pull request by alex-okrushko**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Skeleton for Component Store

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
Side note: We should create a schematic for this, as we've had to this enough times with each new library

File: modules/component-store/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+    # @ngrx/component
```
Comment (timdeschryver) [inline modules/component-store/README.md:1]:
```suggestion
    # @ngrx/component-store
```

File: modules/component-store/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ngrx/component-store",
+  "version": "0.0.0-PLACEHOLDER",
+  "description": "Reactive store for components state",
```
Comment (timdeschryver) [inline modules/component-store/package.json:4]: Like this?

```suggestion
  "description": "Reactive store for component state",
```

**Comment by timdeschryver** [comment]
@alex-okrushko can you also add it to the [intro page](https://github.com/ngrx/platform/blob/a9c4dbb4c2ac0ae5e9cfe81fce156f1cacc5c591/projects/ngrx.io/content/marketing/docs.md#packages)?

**Comment by alex-okrushko** [comment]
Thanks @timdeschryver 
I missed the initial comment about it somehow 🤦‍♂️

**Pull request merged**